test/browser_tests/page_components/footer.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function Footer() {
+  this.get = function() {
+    browser.get( '/' );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/d59e16dd6fa8b0ad43e83530dea0a38410d2f97b